release_notes/44-release-notes.adoc

```diff
@@ -0,0 +1,560 @@
+
+In the {product-title-short} portal, navigate to *Platform Configuration -> Clusters -> Discovered clusters* to view the current status of the discovered clusters.
+
+[id="short-lived-api-tokens-for-central_{context}"]
+=== Short-lived API tokens for Central
+
+{product-title-short} 4.4 introduces the option of issuing short-lived API tokens for interaction with the Central API.
+
+You can exchange the short-lived API tokens for an OpenID Connect (OIDC) identity token. This is useful for the authentication and authorization of machines in, for example, CI environments.
+
+In the {product-title-short} portal, navigate to *Platform Configuration -> Integrations -> Machine access configuration* to create a configuration to enable exchanging OIDC identity tokens for short-lived {product-title-short}-issued tokens.
+
+[id="migration-to-stock-red-hat-openshift-sccs-during-manual-upgrade-by-using-roxctl-cli_{context}"]
+=== Migration to stock {osp} SCCs during manual upgrade by using roxctl CLI
+
+With {product-title-short} 4.4, the platform moves away from using custom security context constraints (SCCs). Instead, {product-title-short} services use {osp} SCCs, which ensures future proof operation and a consistent security posture for {product-title-short}.
+
+For more information, see xref:../upgrading/upgrade-roxctl.adoc#migrating-sccs-during-the-manual-upgrade_upgrade-roxctl[Migrating SCCs during the manual upgrade].
+
+[id="notable-technical-changes_{context}"]
+== Notable technical changes
+
+* The default memory requirement for `scanner-db` has been increased from 200MiB to 512MiB to prevent out of memory (OOMs) errors during database initialization when the memory pressure reaches the node.
+
+* Scanner-slim can now read additional certificate authorities (CAs) correctly from the `additional-ca-sensor` secret.
+
+* Administrator access is not required to delegate ad hoc scan requests to secured clusters.
+
+* Existing integrations that are automatically generated are now deleted on Central startup if `ROX_DISABLE_AUTOGENERATED_REGISTRIES` is set to `true`.
+
+* The `/v1/administration/usage` API endpoint is now considered stable.
+
+* By requesting the `central-monitoring-tls` \ `sensor-monitoring-tls` secrets at startup, you can ensure the presence of the {osp} monitoring `/metrics` server certificate. This requirement only applies if {osp} Monitoring is enabled.
+
+* Simplify your configuration by using `ROX_MEMLIMIT` instead of `GOMEMLIMIT` in configuration files. You can now capture your deployment memory limits and easily adjust `GOMEMLIMIT` by using `ROX_MEMLIMIT`. Define `ROX_MEMLIMIT` as an integer representing the number of bytes and simplify your configuration for more efficient deployment.
```

Comment:
I don't think this quite captures the fact that the `GOMEMLIMIT` environment variable is going away and being replaced by `ROX_MEMLIMIT`.  Also I would like to add a sentence about manual roxctl upgrades here. Here are my edits - maybe @RTann can review?

```suggestion
* The `ROX_MEMLIMIT` environment variable replaces the `GOMEMLIMIT` variable in configuration files. You can now capture your deployment memory limits by using the `ROX_MEMLIMIT` variable. Define `ROX_MEMLIMIT` as an integer representing the number of bytes and simplify your configuration for more efficient deployment. If you are manually upgrading {product-title-short} by using the `roxctl` CLI, you must manually edit the deployments to use the new variable. See xref:../upgrading/upgrade-roxctl.adoc#upgrade-roxctl[upgrading/upgrade-roxctl] for more information.
```

Reply:
`GOMEMLIMIT` is a standard Go environment variable, which is still acceptable to use, but we replaced all instances of it in our configuration files with `ROX_MEMLIMIT` so we can do some extra stuff. Maybe we can make it clear that `GOMEMLIMIT` is still ok, but `ROX_MEMLIMIT` is preferred

Reply:
s/ Use the `ROX_MEMLIMIT`environment variable, which replaces the `GOMEMLIMIT` variable in configuration files. Although you can still use `GOMEMLIMIT` as a standard Go environment variable, use `ROX_MEMLIMIT` instead to more efficiently capture the memory limits for your deployment. Define `ROX_MEMLIMIT` as an integer representing the number of bytes to simplify your configuration. If you are manually upgrading {product-title-short} by using the `roxctl` CLI, ensure to edit the deployments manually to use the new variable. 

For more information, see xref:../upgrading/upgrade-roxctl.adoc#upgrade-roxctl[Manually upgrading using the roxctl CLI].


WDYT? @RTann @kcarmichael08

Reply:
> Although you can still use `GOMEMLIMIT` as a standard Go environment variable, use `ROX_MEMLIMIT` instead to more efficiently capture the memory limits for your deployment. Define `ROX_MEMLIMIT` as an integer representing the number of bytes to simplify your configuration.

Although you may still use the standard Go environment variable `GOMEMLIMIT`, use `ROX_MEMLIMIT` instead to more effectively capture the memory limits for your deployment. `ROX_MEMLIMIT` will set the "soft" memory limit of the Go process to 95% of the configured amount. Define `ROX_MEMLIMIT` as an integer without any units representing the number of bytes.